doc/10-icinga-template-library.md

```diff
@@ -190,6 +190,82 @@ sleep\_time     | **Optional.** The duration of the sleep in seconds. Defaults t
 <!-- keep this anchor for URL link history only -->
 <a id="plugin-check-commands"></a>
 
+### ifw-api <a id="itl-ifw-api"></a>
+
+Built-in check command for executing arbitrary PowerShell check commands via the
+[Icinga for Windows REST API](https://icinga.com/docs/icinga-for-windows/latest/doc/110-Installation/30-API-Check-Forwarder/).
+
+That feature lets the PowerShell processes spawned by Icinga just talk
+to the pre-loaded IfW API instead of loading all PowerShell check commands
+by itself on every check. In contrast the `ifw-api` command doesn't even spawn
+any process, but communicates directly with the IfW API.
+
+Ideally you run the Icinga Director and have imported our PowerShell check commands from
+[their basket](https://icinga.com/docs/icinga-for-windows/latest/doc/200-Icinga-Integration/01-Director-Baskets/).
+In this case adding `ifw-api` to the imports of the _PowerShell Base_
+check command is enough to enable `ifw-api` globally.
+
+!!! warning
+
+    Do the latter only if your **entire** Icinga cluster runs v2.14+! Otherwise,
+    this will break all older nodes which load the modified _PowerShell Base_
+    from a global zone while the imported `ifw-api` is missing.
```

Comment:
"will break all older nodes" is quite vague and might sound more dangerous than it is. I'd use a more precise and less scary description here like "older nodes would reject the new configuration".